pkg/datastore/mysql/ensurer/indexes.sql

```diff
@@ -98,13 +98,17 @@ CREATE INDEX event_project_id_updated_at_desc ON Event (ProjectId, UpdatedAt DES
 ALTER TABLE Event ADD COLUMN EventKey VARCHAR(64) GENERATED ALWAYS AS (data->>"$.event_key") VIRTUAL NOT NULL, ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL NOT NULL;
 CREATE INDEX event_key_name_project_id_created_at_desc ON Event (EventKey, Name, ProjectId, CreatedAt DESC);
 
+-- index on `ProjectId` ASC, `Status` ASC, CreatedAt DESC
+ALTER TABLE Event ADD COLUMN Status INT GENERATED ALWAYS AS (data->>"$.status") VIRTUAL NOT NULL;
+CREATE INDEX project_id_status_created_at_desc ON Event (ProjectId, Status, CreatedAt DESC);
```

Comment:
I think it should have the prefix of `event_` to ensure there will be no conflict with other tables' indexes name. (same for the other)

Reply:
You're right